Source/ThirdParty/xatlas/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(xatlas)
 
 # Urho3D: Removed all extra things
 add_library(xatlas STATIC xatlas.cpp)
+set_target_properties(xatlas PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
Can we do that in 3rdparty cmakefile?
Maybe just for-loop in the end.

Reply:
Seems like we can, yes.